src/app/utils/storage.js

```
@@ -0,0 +1,30 @@
+export const STORAGE_KEY = "bookmarkData"
+
+function getData() {
+  if (typeof window === "undefined") return {}
```

```suggestion
```
It should not make the error fail silently here.